routes/users.go

```diff
@@ -203,3 +218,83 @@ func GetUserHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	json.NewEncoder(w).Encode(user)
 }
+
+func PutUpdateUserHandler(w http.ResponseWriter, r *http.Request) {
+
+	// Checks for food truck ID
+	params := mux.Vars(r)
+	userID, userIDExists := params["userID"]
+	if !userIDExists {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	// Get user from context
+	_, userLoggedIn := r.Context().Value(middleware.UserKey).(string)
```

Comment:
So we would use the first value here instead of a route parameter

Reply:
how do u get the userID from the user variable? it's a string right?

Reply:
Check the /profile route

Reply:
remove _ and replace with userID